generators/ci-cd/templates/_Jenkinsfile

```diff
@@ -156,6 +156,27 @@ node {
 <%_ if (jenkinsIntegrations.includes('docker')) { _%>
 <%= gitLabIndent %>    }
 <%_ } _%>
+<%_ if (jenkinsIntegrations.includes('publishDocker')) { _%>
+
+<%= gitLabIndent %>    def dockerImage
+<%= gitLabIndent %>    stage('build docker') {
+    <%_ if (buildTool === 'maven') { _%>
+<%= gitLabIndent %>        sh "cp -R src/main/docker target/"
```

Comment:
Yes. So l propose to use the Jenkins plugin for now and using maven/gradle can be done later as an improvement.